hera_cal/VERSION

```diff
@@ -1 +1 @@
-2.0
+3.0
```

Comment:
I think something like `3.0.0` is better, but I'll leave it up to you.

Reply:
Given our history, I highly doubt we'll ever do that level of versioning.